python/constellation/satellites/LeCroy/__main__.py

```diff
@@ -0,0 +1,33 @@
+"""
+SPDX-FileCopyrightText: 2024 DESY and the Constellation authors
+SPDX-License-Identifier: EUPL-1.2
+
+Provides the entry point for the LeCroy/LeCrunch satellite
+"""
+
+from typing import Any
+
+from constellation.core.base import EPILOG
+from constellation.core.datasender import DataSenderArgumentParser
+from constellation.core.logging import setup_cli_logging
+
+from .LeCroySatellite import LeCroySatellite
+
+
+def main(args: Any = None) -> None:
+    """Satellite controlling a LeCroy oscilloscope"""
+
+    # Get a dict of the parsed arguments
+    parser = DataSenderArgumentParser(description=main.__doc__, epilog=EPILOG)
```

Comment:
```suggestion
    parser = DataSenderArgumentParser(description=main.__doc__)
```

Minor: let's not add the epilog for satellites